apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html

```diff
@@ -1,4 +1,122 @@
+        class="dropdown-toggle" aria-controls="dropdown-alignment">
+        <span class="icon-dots-three-vertical"></span>
+        </button>
+        <ul id="dropdown-alignment" *dropdownMenu
+        class="dropdown-menu dropdown-menu-right bottom-details pt-1 pb-1" role="menu"
+        aria-labelledby="button-alignment">
+            <li role="menuitem">
+                <section
+                    *ngIf="groupStockReport && groupStockReport.stockReport && groupStockReport.stockReport.length>0">
+                        <div class="p-3">
+                            <div class="row mr-0 ml-0">
+        
+                                <div class="col-sm-5">
+                                    <div class="row mr-0 ml-0">
+                                        <div class="col-xs-5 p0 count-text">
+                                            <h4 class="text-light pl-0">
+                                                <div *ngIf="GroupStockReportRequest.branchDetails==='allEntity'">Transaction
+                                                    Count</div>
+                                                <div class="" *ngIf="GroupStockReportRequest.branchDetails!=='allEntity'">
+                                                    Product Count</div>
+                                                <div class="mt-1">Profit/Loss</div>
+                                            </h4>
+                                        </div>
+                                        <div class="col-xs-7 p0">
+                                            <h4 class="text-orange text-right">
+                                                <div *ngIf="GroupStockReportRequest.branchDetails==='allEntity'">
+                                                    <span class="text-orange"
+                                                        *ngIf="groupStockReport.totalItems">&nbsp;{{groupStockReport.totalItems}}</span>
+                                                </div>
+                                                <div class="" *ngIf="GroupStockReportRequest.branchDetails!=='allEntity'">
+                                                    <span class="text-orange"
+                                                        *ngIf="groupStockReport.totalItems">&nbsp;{{groupStockReport.totalItems}}</span>
+                                                </div>
+                                                <div class="mt-1">
+                                                    <span class="text-orange d-block">{{groupStockReport.profit | giddhCurrency}}
+                                                    </span>
+                                                </div>
+                                            </h4>
+                                        </div>
+                                    </div>
+                                    <div class="row" *ngIf="groupStockReport.openingBalance.transactionCount">
```

Comment:
Do use safe navigation operator for accessing all the nested object keys for eg `groupStockReport`